### DISCUSSION TITLE
鼠标单击实现 暂停/播放

### DISCUSSION THREAD

**Discussion by Lockik** [diana7127/mpv.net-DW#22] [Ideas]
可不可以添加一个  鼠标单击实现 暂停/播放

**Comment by MC-dusk** [comment]
ctrl + i 打开快捷键列表，添加命令`cycle pause`，快捷键设定进去直接点鼠标就能识别输入，或者手动写`MBTN_LEFT`

**Comment by Lockik** [comment]
我试了 提示重启生效  但是还是不行 设置里面显示了MBTN_LEFT  但是左键单击还是无法实现播放/暂停

**Reply by MC-dusk** [reply]
我是可以的，你可能快捷键冲突了，或者不是最新版本

**Reply by MC-dusk** [reply]
看起来没问题
我还在用老版本，所以新版本可能真不行（
另外虽然我之前为了测试试过改成单击，但实际使用还是用的双击暂停播放
